pandas/core/groupby/groupby.py

```diff
@@ -190,14 +190,13 @@ class providing the base-class of operations.
 )
 
 _pipe_template = """
-Apply a function `func` with arguments to this %(klass)s object and return
-the function's result.
+Apply a function `func` with arguments to this %(klass)s object and return the result.
 
 %(versionadded)s
 
 Use `.pipe` when you want to improve readability by chaining together
 functions that expect Series, DataFrames, GroupBy or Resampler objects.
-Instead of writing
+Instead of writing.
```

Comment:
Is this full stop necessary here? It doesn't feel like it's the end of the sentence, could it be changed to "Instead of writing:"?

Reply:
Okay